### ISSUE TITLE
4.03 Error when handling VFR videos

### ISSUE THREAD

**Issue by Last-Order** [rigaya/NVEnc#64]
I'm trying to use `avsync forcecfr --fps 30` to convert a VFR video to CFR but got

```
timestamp not acquired successfully from input steram, avsync vfr cannot be used.
  [0x21] PTS_DUPLICATE
Failed to open input file.
```

My encoding options:
```
NVEncC64.exe -i "F:\vfr.mp4" --avhw --audio-codec --audio-bitrate 192 --avsync forcecfr --fps 30 -o "F:\cfr2.mp4" --cqp 21
```

Here is the mediainfo of the video file
```
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Main@L3.1
Format settings, CABAC                   : Yes
Format settings, ReFrames                : 4 frames
Format settings, GOP                     : M=4, N=15
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 4 min 49 s
Bit rate                                 : 1 987 kb/s
Width                                    : 800 pixels
Height                                   : 450 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 30.000 FPS
Minimum frame rate                       : 29.412 FPS
Maximum frame rate                       : 30.303 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.184
Stream size                              : 68.6 MiB (91%)
Color range                              : Limited
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709

ID                                       : 2
Format                                   : AAC
Format/Info                              : Advanced Audio Codec
Format profile                           : LC
Codec ID                                 : 40
Duration                                 : 4 min 49 s
Bit rate mode                            : Constant
Bit rate                                 : 186 kb/s
Channel(s)                               : 2 channels
Channel positions                        : Front: L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 46.875 FPS (1024 spf)
Compression mode                         : Lossy
Stream size                              : 6.41 MiB (9%)
Default                                  : Yes
Alternate group                          : 1
```

And here is the file (Google Drive):
https://drive.google.com/open?id=1uDQbgUYU1gyYYxDlpE9VxfbZK3jY9B6W

**Comment by rigaya** [comment]
Thank you for providing source file, should be fixed with [NVEnc 4.11](https://github.com/rigaya/NVEnc/releases/tag/4.11).

**Comment by Last-Order** [comment]
Thanks for your quick reply and fix.
